CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 ### Fixed
 
 - Rapidjson assert on invalid syntax for first vehicle (#418)
+- Compatibility with Visual Studio 2019 (#448)
```

Comment:
Now that we have a `release/1.9` branch, this should appear in a new `Unreleased` entry. I'll push the usual technical commit on `master` to allow this.

Reply:
just realized I forgot smth here.. I can do it as well then.

Reply:
done